admin/run-python3-tests

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3.3
+#!/usr/bin/env python3
```

Comment:
This change is unrelated to this branch; would you mind making it in a separate PR?